packages/docs/docs/getting-started/flow.mdx

```diff
@@ -98,36 +98,128 @@ function. Generators like this are called tweens, because they animate
 be**tween** two values. You can read more about them in the
 [tweening](/docs/tweening) section.
 
+## Flow Generators
+
 Another kind of generators are _flow generators_. They take one or more
 generators as their input and combine them together. We've mentioned the `all()`
 generator in the quickstart section, there's a few more:
 
-## `all`
+### `all`
 
 <ApiSnippet url={'/api/core/flow#all'} />
 <hr />
 
-## `any`
+### `any`
 
 <ApiSnippet url={'/api/core/flow#any'} />
 <hr />
 
-## `chain`
+### `chain`
 
 <ApiSnippet url={'/api/core/flow#chain'} />
 <hr />
 
-## `delay`
+### `delay`
 
 <ApiSnippet url={'/api/core/flow#delay'} />
 <hr />
 
-## `sequence`
+### `sequence`
 
 <ApiSnippet url={'/api/core/flow#sequence'} />
 <hr />
 
-## `loop`
+### `loop`
 
 <ApiSnippet url={'/api/core/flow#loop'} />
 <hr />
+
+## Looping
+
+There are many ways to animate multiple objects. Here are some examples. Try
+using them in the below editor.
+
+```tsx editor
```

Comment:
You can use `ratio` to make the preview a bit higher so that the squares don't overflow it:
```suggestion
```tsx editor ratio=2
```